src/main/java/com/robinpowered/sdk/model/User.java

```diff
@@ -163,4 +163,16 @@ public Reference(int userId) {
             super(userId);
         }
     }
+
+    public static class DeviceId {
+        private Integer deviceId;
+
+        public DeviceId(Integer deviceId) {
+            this.deviceId = deviceId;
+        }
+
+        public Integer getDeviceId() {
+            return deviceId;
+        }
+    }
```

Comment:
This allows us to use the `User.DeviceId` as the body for event confirmation requests.

Due to [this bug](https://github.com/robinpowered/robin-java-sdk/pull/21/files#diff-4fdbdff9f7f51b6bf5d246436ee3ac66R51) in 1.x versions of Retrofit, we need to include a non-null body in each request.